src/effects.js

```diff
@@ -0,0 +1,18 @@
+var old = jQuery.Tween.prototype.run;
```

Comment:
It would be good to use a more unique name like `oldRun` here. If another module accidentally declared `old` it should be found by JSHint but it might be confusing to figure it out.